inc/ec_base.h

```diff
@@ -228,11 +228,74 @@ typedef struct {
       } creds;
 } ec_credential_object_t;
 
+/**
+ * @brief A DPP attribute as defined in EC 8.1. Can be sent/received over the network.
```

Comment:
In header files when referencing specifications, can you spell out the spec (EasyConnect) instead of abreviating (EC)
```suggestion
 * @brief A DPP attribute as defined in EasyConnect 8.1. Can be sent/received over the network.
```